lib/src/util/react_wrappers.dart

```diff
@@ -281,7 +281,7 @@ react.Component getDartComponent(/* ReactElement|ReactComponent|Element */ insta
     return null;
   }
 
-  assert(() {
```

Comment:
Was passing functions to `assert` removed in Dart 2 or something?

Reply:
@greglittlefield-wf the assert arg has to return a bool in dart 2.